src/js/grapl-cdk/lib/dgraph.ts

```diff
@@ -40,13 +40,13 @@ class Zero extends cdk.Construct {
         zeroTask.addContainer(id + 'Container', {
             // --my is our own hostname (graph + id)
             // --peer is the other dgraph zero hostname
-            image: ecs.ContainerImage.fromRegistry("dgraph/dgraph"),
+            image: ecs.ContainerImage.fromRegistry("dgraph/standalone:v1.2.2"),
```

Comment:
I don't think we want to use standalone. Standalone is not clustered or intended for production.

Reply:
changed to `dgraph/dgraph:v1.2.2`